### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/controller/telemetry/bpf_prog_metric.go [hunk 1]
```diff
@@ -117,49 +116,45 @@ func (m *OperationMetricController) Run(ctx context.Context, KmeshPerfInfo *ebpf
...
+		operationType: operationTypeMap[data.operationType],
 	}
-	labels.nodeName = nodeName
-	labels.operationType = operationTypeMap[data.operationType]
-	return labels
 }
 
-func (m *OperationMetricController) updateOperationMetricCache(data operationUsageInfo, labels operationMetricLabels) {
+func (m *BpfProgMetric) updateOperationMetricCache(data operationDuration, labels operationMetricLabels) {
+	m.mutex.Lock()
```
Comment (YaoZengzeng) [inline pkg/controller/telemetry/bpf_prog_metric.go:139]: Lock twice?

Comment (hzxuzhonghu) [reply pkg/controller/telemetry/bpf_prog_metric.go:139]: Oh, i meant to move lock into hte function

File: pkg/controller/controller.go [hunk 2]
```diff
@@ -108,10 +108,10 @@ func (c *Controller) Start(stopCh <-chan struct{}) error {
 			return fmt.Errorf("fail to start ringbuf reader: %v", err)
 		}
 	}
-	c.client = NewXdsClient(c.mode, c.bpfAdsObj, c.bpfWorkloadObj, c.enableAccesslog)
+	c.client = NewXdsClient(c.mode, c.bpfAdsObj, c.bpfWorkloadObj, c.enableAccesslog, c.enablePerfMonitor)
```
Comment (LiZhenCheng9527) [inline pkg/controller/controller.go:111]: We should add the ability of kmeshctl to control the bpf perf monitor

Comment (hzxuzhonghu) [reply pkg/controller/controller.go]: SGTM

**Comment by hzxuzhonghu** [comment]
@nlgwcy ptal

**Pull request merged**